### ISSUE TITLE
Segregate Alerts for 5xx and 4xx error codes

### ISSUE THREAD

**Issue by aliasgerkw** [checkly/public-roadmap#80]
💡 For general support requests and bug reports, please go to [checklyhq.com/support](https://checklyhq.com/support)

**Is your feature request related to a problem? Please describe.**
Currently 4xx alerts are coming due to some mistake in request params or body. Want to avoid audience to be on toes for such alerts ended informing dev team to fix the check rather checking application

**Describe the solution you'd like**
Alert channel can have additional filter to mute 4xx alerts not to go to slack and only to registered emails which opt for them.

**Describe alternatives you've considered**
Currently we are ignoring those and reducing that from outage count.

**Additional context**
NA

**Comment by tnolet** [comment]
Hi Aliasger, thanks for submitting. I don't have a clear workaround for this, but the situation seems very specific. Adding a filter for 4xx errors on alerting channels is probably not what we want here as it would just tackle this one situation while making the application more complex and confusing.

**Comment by aliasgerkw** [comment]
Yeah situation is more specific to us but can help other customers who actively faced 4xx during application or check changes.
